include/ringmesh/basic/types.h

```diff
@@ -82,12 +96,33 @@ namespace RINGMesh
         return true;
     }
 
+    template < index_t DIMENSION >
+    bool operator==(
```

Comment:
Is it possible to avoid code duplication by templating vec, ivec, sivec? Or an other strategy?

Reply:
Tried something by templating it from the original vector GEO::vecng, it seems to work.